generics/helpers/solutionAndProjectTemplateUtils.js

```diff
@@ -114,7 +116,7 @@ function createSolution(solutionData, checkDate = false, userDetails) {
 			programMatchQuery['_id'] = solutionData.programId
 
 			let updateProgram = await programQueries.findAndUpdate(programMatchQuery, {
-				$addToSet: { components: solutionCreation._id },
+				$addToSet: { components: { _id: solutionCreation._id, order: programsComponent.length + 1 } },
```

Comment:
@borkarsaish65 while creating solution we don't have to expect any order passed for the solution